src/EditorFeatures/CSharpTest/UseInferredMemberName/UseInferredMemberNameTests.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.UseInferredMemberName;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.InferredMemberName
+{
+    [Trait(Traits.Feature, Traits.Features.CodeActionsUseInferredMemberName)]
+    public class UseInferredMemberNameTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
+    {
+        internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProviderAndFixer(Workspace workspace)
+            => (new CSharpUseInferredMemberNameDiagnosticAnalyzer(), new CSharpUseInferredMemberNameCodeFixProvider());
+
+        private static readonly CSharpParseOptions s_parseOptions =
+            CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp7_1);
```

Comment:
So, i would prefer you remove this.  IDE test style is to have our tests basically always runnign against the latest stuff.  For tests that actually have to validate specific version beahvior, we make those tests special.  i.e. validating that something is off in an old version.